### DISCUSSION TITLE
Question about mean_en.i file

### DISCUSSION THREAD

**Discussion by JaeHyun-Nam** [shannon-lab/zapdos#95] [Q&A Plasma Chemistry / CRANE] [answered]
Hello, I'm a prospective Ph.D. student and a beginner in this interesting field.

While studying Zapdos application and the example "mean_en.i" file, I found that Ar+ is solved in the domain 0 (gas) and OH^- is solved in the domain 1(liquid), but I couldn't find any variables of positive ions in the liquid. How this could be possible? This does not seem to make sense physically.

Anyway, are there any materials that contain the details of this example?

**Comment by lindsayad** [comment, marked as answer]
This is a relatively crude chemistry model. But this is a DC discharge and the results illustrate that current is driven in the following way:

Near cathode (left side of the domain): Ar+
Bulk gas: electrons (g)
Gas side of interface: electrons (g)
Liquid side of interface: electrons (aq)
Moving into bulk/towards anode: electrons/OH- with ratio skewing towards OH- the farther you move away from the interface

This input is focused mostly on electron dynamics and on determining the primary charge carriers. We only consider two reactions in this input:

e- + H2O -> H + OH-
2e- + 2H2O -> H2 + 2OH-

If we were to switch the polarity, then we would need to model cations in the liquid because the primary flux coming from the plasma into the liquid would be the cations.

The best documentation for this particular input is [this article](https://iopscience.iop.org/article/10.1088/0022-3727/49/23/235204/meta) although I am ashamed to admit that I didn't document the two aqueous phase reactions I was modeling!
